source/MaterialXGenMdl/mdl/materialx/cm.mdl

```diff
@@ -0,0 +1,245 @@
+//
+// Copyright Contributors to the MaterialX Project
+// SPDX-License-Identifier: Apache-2.0
+//
+// Default color management transforms for MDL
```

Comment:
Although I see some good proposed changes in this pull request, I think the color management MDL file may be from an _earlier_ version of MaterialX, and that it was removed when we implemented the color management library as language-independent graphs:

https://github.com/AcademySoftwareFoundation/MaterialX/pull/1352

Reply:
Thank you for the review and giving the history. Removed this file.